generate-task-list.js

```diff
@@ -0,0 +1,55 @@
+const addTaskForm = document.querySelector('.task-form');
+const taskList = document.querySelector('.task-list');
+const addTask = event => {
+	event.preventDefault();
+
+	const taskName = document.querySelector('.task-form__input--task-content').value;
+	const taskDeadline = document.querySelector('.task-form__input--date').value;
+
+	let lastElement = null;
+	let lastElementIndex = null;
+
+	console.log(tasks.length);
+	if (tasks.length > 0) {
+		lastElementIndex = tasks.length - 1;
+		lastElement = tasks[lastElementIndex].name;
+	} else {
+		lastElement = tasks[0];
+		lastElementIndex = 0;
+	}
+	tasks.push({
+		id: lastElement.id + 1,
```

Comment:
Instead of integers for `id`s, use [uuid](https://www.npmjs.com/package/uuid#quickstart---browser-ready-versions) package (no need to do the `lastElementIndex` magic then).